auth.go

```diff
@@ -27,13 +31,14 @@ func (c *Client) renewToken() {
 	jsonToken := make(map[string]string)
 
 	for {
+		renewalLoopRunning = true
```

Comment:
Does this need to toggle back to `false` at some point?

Reply:
no. could probably move it out of the loop